assets/scss/components/_avatar.scss

```diff
@@ -44,7 +44,7 @@ Class                     | Description
 `avatar--noPhoto`         | Resets `text-indent` for avatars without photos, allowing display of fallback content
 */
 $noPhotoIconOpacity: alpha($C_textHint);
-$avatarBg: #E4E9ED; // change to $C_coolGrayLight once we start using `swarm-constants`
+$avatarBg: $C_coolGrayLight;
```

Comment:
Not really part of this PR, but I figured I'd update SASS vars after upgrading to a version of `swarm-sasstools` that pulls from `swarm-constants`